src/content/tutorial/wrapping-up/wrapping-up.mdx

```diff
@@ -0,0 +1,62 @@
+
+<AnchorLinks>
+
+- [Badging](#badging)
+- [Free t-shirt](#free-t-shirt)
+- [Application](#application)
+
+</AnchorLinks>
+
+## Badging
+
+Once you have completed all five steps of the tutorial, you can apply for the IBM Digital Badge.
+
+<ImageComponent cols="8" caption="IBM Digital Badge">
+
+![](/images/carbon-badge.png)
+
+</ImageComponent>
+
+### IBM Design System React Components
+
+This badge earner demonstrates knowledge about Carbon's React components. What it takes to earn the badge:
+
+1. Completion of steps 1 through 5 of the React Carbon tutorial
+   - Step 1. [Installing Carbon](/tutorial/react-step-1)
+   - Step 2. [Building pages](/tutorial/react-step-2)
+   - Step 3. [Using APIs](/tutorial/react-step-3)
+   - Step 4. [Creating components](/tutorial/react-step-4)
+   - Step 5. [Deploying to IBM Cloud](/tutorial/react-step-5)
+2. Submission of _approved_ pull requests for steps 1 through 5 of the React Carbon tutorial in the [carbon-tutorial repository](https://github.com/carbon-design-system/carbon-tutorial).
+
+### FAQ and help
+
+_Badge Issuer Contact:_ kellychurchill@us.ibm.com
+
+_Acclaim Support:_ For questions related to your Acclaim badge earner account and profile, as well as issues related to claiming your badge after receiving a notification, go to [http://support.youracclaim.com](http://support.youracclaim.com) for account issues.
+
+### Privacy statement
+
+NOTICE: IBM leverages the services of Credly's Acclaim platform, a 3rd party data processor authorized by IBM and located in the United States, to assist in the administration of the IBM Digital Badge program. In order to issue you an IBM Digital Badge, your personal information (name, email address, and badge earned) will be shared with the Credly's Acclaim platform. You will receive an email notification from Acclaim with instructions for claiming the badge. Your personal information is used to issue your badge and for program reporting and operational purposes. It will be handled in a manner consistent with IBM privacy practices. The IBM Privacy Statement can be viewed here: [https://www.ibm.com/privacy/us/en/](https://www.ibm.com/privacy/us/en/). IBM employees can view the IBM Internal Privacy Statement here: [https://w3.ibm.com/w3publisher/w3-privacy-notice](https://w3.ibm.com/w3publisher/w3-privacy-notice).
+
+## Free t-shirt
+
+Submit your completed IBM Digital Badge application by July 30, 2019, and receive a free\* Carbon t-shirt.
+
+<ImageComponent cols="8" caption="Carbon t-shirt, #swag #threads #highfashion">
+
+![](/images/carbon-tee.png)
+
+</ImageComponent>
+
+## Application
+
+Complete the [application form]() to earn the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

Comment:
`to earn the digital badge and t-shirt` should say `to request the digital badge and t-shirt`